cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml

```diff
@@ -36,7 +36,7 @@ spec:
         hostPath:
           path: /dev
       containers:
-      - image: "gcr.io/gke-release/nvidia-gpu-device-plugin@sha256:a75ec0caa9e3038bd9886b3f36641a624574ff34b064974de6ee45048de3372b"
+      - image: "us.gcr.io/k8s-artifacts-prod/nvidia-gpu-device-plugin@sha256:4b036e8844920336fa48f36edeb7d4398f426d6a934ba022848deed2edbf09aa"
```

Comment:
can we use `k8s.gcr.io` instead of `us.gcr.io/k8s-artifacts-prod/`?

Reply:
Yes. The registry you have here is one of the backing registries for that.

Reply:
Can we put an advisory tag here?
:Foo@sha256 works and ignores foo but tells the reviewer what version you intended to pin
What's the delta on this image other than the registry?

Reply:
there is no tag that is easily human-readable :(
https://console.cloud.google.com/gcr/images/k8s-artifacts-prod/US/nvidia-gpu-device-plugin?project=k8s-artifacts-prod&organizationId=758905017065&gcrImageListsize=200

I use the same that was used here https://github.com/kubernetes/kubernetes/pull/96276

Do you know who is the best person to clarify?

Reply:
we should probably be using k8s.gcr.io if we're doing this.
i'm still not sure if this image is safe for us to be hosting 🤔

Reply:
i'm not sure who can clarify, SIG node would probably be the right place to start looking.

Reply:
Agreed that we should be using k8s.gcr.io/... here instead.
Also, this image reference is top-level, implying it may be a backfill?

When's the last time this image was pushed?

Reply:
`  "us.gcr.io/k8s-artifacts-prod/nvidia-gpu-device-plugin@sha256:4b036e8844920336fa48f36edeb7d4398f426d6a934ba022848deed2edbf09aa"` GCP says it is from `May 30, 2019` -> https://console.cloud.google.com/gcr/images/k8s-artifacts-prod/US/nvidia-gpu-device-plugin?gcrImageListsize=30&gcrImageListquery=%255B%257B_22k_22_3A_22_22_2C_22t_22_3A10_2C_22v_22_3A_22_5C_224b036e8844920336fa48f36edeb7d4398f426d6a934ba022848deed2edbf09aa_5C_22_22%257D%255D

there is a newer image from `Mar 16, 2020` - `us.gcr.io/k8s-artifacts-prod/nvidia-gpu-device-plugin@sha256:ebd0d26fb21ae9c7d11da156737006ceff085029c697a3a163774188c3328dbe` https://console.cloud.google.com/gcr/images/k8s-artifacts-prod/US/nvidia-gpu-device-plugin?gcrImageListsize=30&gcrImageListquery=%255B%257B_22k_22_3A_22_22_2C_22t_22_3A10_2C_22v_22_3A_22_5C_2297dad60fbfc0feab9784d2ce1f2578e8fd1050aa_5C_22_22%257D%255D


I will update the PR with this new image and ask for help in the slack sig-node channel

Reply:
slack post: https://kubernetes.slack.com/archives/C0BP8PW9G/p1623920550355600